backend/corpora/common/utils/corpora_constants.py

```diff
@@ -0,0 +1,26 @@
+class CorporaConstants(object):
+
+    # Constants related to the metadata requirements of submitting a dataset to Corpora.
+    REQUIRED_OBSERVATION_METADATA_FIELDS = ["TISSUE", "ASSAY", "DISEASE", "CELL_TYPE", "SEX", "ETHNICITY"]
```

Comment:
We have a new one: `development_stage`

Reply:
I lumped that into the second validation PR: https://github.com/chanzuckerberg/corpora-data-portal/pull/383.

Reply:
👍